glide.yaml

```diff
@@ -2,16 +2,18 @@ package: github.com/kubeless/kubeless
 import:
 - package: github.com/sirupsen/logrus
 - package: github.com/spf13/cobra
+  version: v0.0.1
 - package: k8s.io/apimachinery
-  version: 9d38e20d609d27e00d4ec18f7b9db67105a2bde0
+  version: kubernetes-1.8.7
 - package: k8s.io/client-go
+  version: kubernetes-1.8.7
   subpackages:
     - plugin/pkg/client/auth/gcp
   version: v5.0.0
 - package: k8s.io/api
-  version: fe29995db37613b9c5b2a647544cf627bfa8d299
+  version: kubernetes-1.8.7
 - package: k8s.io/code-generator
-  version: master
+  version: kubernetes-1.8.7
 - package: k8s.io/gengo
   version: master
```

Comment:
can you also pin `k8s.io/gengo` version ? That's what I wanted to do in #567

Reply:
@ngtuna unfortunately, there are no releases for `kubernetes/gengo` https://github.com/kubernetes/gengo/releases, hence I kept it to `master`. Can you suggest how can we decide on a particular commit SHA?

Reply:
so just pick up the current sha in your master branch (found in glide.lock). That will be safe:

```
- name: k8s.io/gengo
  version: 1ef560bbde5195c01629039ad3b337ce63e7b321
```

Reply:
Oh hold on... please also test if it works with the new revision `kubernetes-1.8.7` of other k8s.io packages. Otherwise, I am fine with the new revision in this PR but please pin it in glide.yaml

Reply:
Pinned it to the working version